php/Terminus/Collections/SshKeys.php

```diff
@@ -2,6 +2,8 @@
 
 namespace Terminus\Collections;
 
+use Terminus\Exceptions\TerminusException;
```

Comment:
Nope, the command should work. Can you paste the error you're getting and the command you're running?